src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -5354,7 +5364,8 @@ private bool IsConstructorAccessible(MethodSymbol constructor, ref CompoundUseSi
                     diagnostics,
                     out MemberResolutionResult<MethodSymbol> memberResolutionResult,
                     out ImmutableArray<MethodSymbol> candidateConstructors,
-                    allowProtectedConstructorsOfBaseType: false))
+                    allowProtectedConstructorsOfBaseType: false) &&
+                !type.IsAbstract)
```

Comment:
>!type.IsAbstract) [](http://example.com/codeflow?start=16&length=17)

Is this change related to the PR?

Reply:
> Is this change related to the PR?

Yes, it allows us to avoid the wrapping at the end of this ```if``` body. See the removed code. We don't want to end up with a bad expression on top of ```BoundObjectCreationExpression```. We should have either one node, or the other.